packages/app-project/src/screens/ClassifyPage/ClassifyPageContainer.js

```diff
@@ -15,14 +16,9 @@ function ClassifyPageContainer ({
   but can be reset by the Classifier component via onSubjectReset().
   This state does not change via components of the prioritized subjects UI (Next/Prev buttons) */
   const [selectedSubjectID, setSelectedSubjectID] = useState(subjectID)
+  const assignedWorkflowLevel = useAssignedLevel(assignedWorkflowID)
 
-  let assignedWorkflow
   let allowedWorkflows = workflows.slice()
-  let assignedWorkflowLevel = 1
-  if (assignedWorkflowID) {
-    assignedWorkflow = workflows.find(workflow => workflow.id === assignedWorkflowID)
-    assignedWorkflowLevel = parseInt(assignedWorkflow?.configuration.level, 10)
```

Comment:
`parseInt` here makes sure that levels are always compared as numbers, even though the workflow config stores strings.

Reply:
Agreed, however there wasn't a `parseInt` in the original WorkflowSelectButtons lock / unlock sorting, so I didn't initially include it in `useAssignedLevel()`. I'll add it to the hook before merging.